pest/src/parser_state.rs

```diff
@@ -61,10 +61,8 @@ static CALL_LIMIT: AtomicUsize = AtomicUsize::new(0);
 ///
 /// # Arguments
 ///
-/// * `limit` - The maximum call depth or the number of calls.
-///             If 0, the call depth or the number of calls is unlimited.
-/// * `finite_calls` - If `true`, the parser execution is limited to
-///                    a finite number of calls (based on `limit`).
+/// * `limit` - The maximum number of calls. If None,
+///             the number of calls is unlimited.
```

Comment:
That's better thanks. I wonder if you could clarify in the doc comment whether "number of calls" means the same as call depth or is just a running total. Also, please define what constitutes a "call"?

Reply:
After https://github.com/pest-parser/pest/pull/684#issuecomment-1210456883 , it's a running total. I've expanded the doc comment.